### PR TITLE
[6.4] FIX UI test_smartclassparameter

### PR DESCRIPTION
**Pull request by ldjebran**
```
(robottelo) dlezz@dlezz:~/projects/robottelo-fork$ pytest -v tests/foreman/ui_airgun/test_smartclassparameter.py 
============================================ test session starts =============================================
platform linux -- Python 3.6.6, pytest-3.6.1, py-1.5.4, pluggy-0.6.0 -- /home/dlezz/.pyenv/versions/3.6.6/envs/robottelo/bin/python
cachedir: .pytest_cache
shared_function enabled - ON - scope: ffff - storage: file
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: wait-for-1.0.9, xdist-1.22.5, services-1.2.1, mock-1.10.0, forked-0.2, cov-2.5.1
collecting 9 items                                                                                           2018-09-19 16:49:20 - conftest - DEBUG - BZ deselect is disabled in settings

collected 9 items                                                                                            

tests/foreman/ui_airgun/test_smartclassparameter.py::test_positive_create_matcher_attribute_priority PASSED [ 11%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_positive_create_matcher_merge_override PASSED [ 22%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_negative_create_matcher_merge_override PASSED [ 33%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_positive_create_matcher_merge_default PASSED [ 44%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_negative_create_matcher_merge_default PASSED [ 55%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_positive_create_matcher_avoid_duplicate PASSED [ 66%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_negative_create_matcher_avoid_duplicate PASSED [ 77%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_positive_view_yaml_output_after_resubmit_array_type PASSED [ 88%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_positive_view_yaml_output_after_resubmit_yaml_type PASSED [100%]

========================================= 9 passed in 760.24 seconds =========================================
```

### CODE REVIEW

File: tests/foreman/ui_airgun/test_smartclassparameter.py [hunk 1]
```diff
@@ -33,16 +33,24 @@ def module_org():
     return entities.Organization().create()
 
 
+@fixture(scope='module')
+def module_loc():
+    return entities.Location(id=2).read()
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_smartclassparameter.py]: pls use DEFAULT_LOC_ID if you don't mind

File: tests/foreman/ui_airgun/test_smartclassparameter.py [hunk 2]
```diff
@@ -33,16 +33,24 @@ def module_org():
...
 @fixture(scope='module')
-def puppet_env(content_view):
-    return entities.Environment().search(
-        query={'search': u'content_view="{0}"'.format(content_view.name)})[0]
+def puppet_env(content_view, module_org):
+    env = entities.Environment().search(
+        query={'search': u'content_view="{0}" and organization_id={1}'.format(
+            content_view.name, module_org.id)}
+    )[0]
+    return env
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_smartclassparameter.py]: why setting single-use variable instead of just returning it?

Comment (ldjebran) [reply tests/foreman/ui_airgun/test_smartclassparameter.py]: do not like to return directly function call results especially those ones that make external services calls with list access retrieval, but this is not a blocker for me if you insist I can change it :)

**Comment by ldjebran** [comment]
```
(robottelo) dlezz@dlezz:~/projects/robottelo-fork$ pytest -v tests/foreman/ui_airgun/test_smartclassparameter.py
============================================ test session starts =============================================
platform linux -- Python 3.6.6, pytest-3.6.1, py-1.5.4, pluggy-0.6.0 -- /home/dlezz/.pyenv/versions/3.6.6/envs/robottelo/bin/python
cachedir: .pytest_cache
shared_function enabled - ON - scope: ffff - storage: file
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: wait-for-1.0.9, xdist-1.22.5, services-1.2.1, mock-1.10.0, forked-0.2, cov-2.5.1
collecting 9 items                                                                                           2018-09-24 10:18:53 - conftest - DEBUG - BZ deselect is disabled in settings

collected 9 items                                                                                            

tests/foreman/ui_airgun/test_smartclassparameter.py::test_positive_create_matcher_attribute_priority PASSED [ 11%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_positive_create_matcher_merge_override PASSED [ 22%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_negative_create_matcher_merge_override PASSED [ 33%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_positive_create_matcher_merge_default PASSED [ 44%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_negative_create_matcher_merge_default PASSED [ 55%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_positive_create_matcher_avoid_duplicate PASSED [ 66%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_negative_create_matcher_avoid_duplicate PASSED [ 77%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_positive_view_yaml_output_after_resubmit_array_type PASSED [ 88%]
tests/foreman/ui_airgun/test_smartclassparameter.py::test_positive_view_yaml_output_after_resubmit_yaml_type PASSED [100%]

========================================= 9 passed in 646.41 seconds =========================================
```

**Comment by ldjebran** [comment]
@abalakh changes made

**Pull request merged**